### ISSUE TITLE
Outlined music-note Icon does not exist in real life

### ISSUE THREAD

**Issue by ABausG** [google/material-design-icons#817]
The Outlined Version of the Music note and all other icons using a note does not make sense from a musical standpoint.

It shows a half note/minim with a flag.

List of "valid" musical notes
https://en.wikipedia.org/wiki/List_of_musical_symbols#Notes_and_rests

Outlined version of the note icon
https://material.io/tools/icons/?search=music&style=outline

**Comment by lennart-k** [comment]
That's true but I think as long as you don't want to write music with the Material Design Icons we can still use it.
Imo it looks more consistent in combination with other icons.
